plugins/lua/zone.lua

```diff
@@ -1216,10 +1216,188 @@ function RetireLocationOverlay:confirm_retire()
         curry(retire, location))
 end
 
+-- -------------------
+-- AnimalActionsWidget
+--
+
+local YES = 'Yes'
```

Comment:
These constants are of questionable utility

Reply:
Removed constants